test/test_instance_types_service.py

```diff
@@ -102,7 +102,7 @@ def test_instance_types_get(self):
             self.provider,
             "instance_type")
         inst_type = self.provider.compute.instance_types.find(
-            name=instance_type_name)[0]
```

Comment:
Done.
Reverted the changes in test_instance_types_service.py and fixed the docstring.